hyp3_metadata/templates/readme.md.txt.j2

```diff
@@ -36,9 +36,13 @@ The source granule used to generate the products contained in this folder is:
 Please refer to the ASF Sentinel-1 RTC User Guide for in-depth guidance on the use of this dataset:
 * https://asf.alaska.edu/wp-content/uploads/2019/02/Sentinel_RTC_Users_Guide.pdf
 
-When using this data in a publication or presentation, we ask that you include the following acknowledgement:
+When using this data as an image in a publication such as journal papers, articles, presentations, posters, and websites, please include the following credit with the image (portions in square brackets are optional):
 
-    RTC product processed by ASF DAAC HyP3 {{ processing_date.year }} using GAMMA software. Contains modified Copernicus Sentinel data {{ granule_name[17:21] }}, processed by ESA.
+    [RTC product processed by ]ASF DAAC HyP3 {{ processing_date.year }}[ using GAMMA software]. Contains modified Copernicus Sentinel data {{ granule_name[17:21] }}, processed by ESA.
```

Comment:
I think I would just drop the second square bracket here (and maybe both?) -- the way you explain the more detailed one is perfect, and user can decide to include that part or not. 

If you want the second square bracket here, I'd generalize it to `[ using {{ processor_name }} software]` here.

Reply:
Thanks for the processor_name tip. I've made that change.

I'd still like to keep the processor in the image credit, at least as an option. I think it is useful information to include when distributing an image. But I'll check in with @talogan and @khogenso as well.

As for the xml files, because they're directed at the GIS community, I would think that the image credit would be of generally of more use, especially for the actual RTC products. There's no reason we can't include the full text, as in the readme, but if I was only going to include one, I think I would err on the side of the image credit. 

I will push some changes using a credit statement in the product xml that is similar to what's currently in the readme, and solicit feedback on that. 

For the auxiliary files, I think I'm okay with leaving them as they are (the acknowledgement with the versions included). Because these files are not likely to be used as illustrations, this one covers all the bases. We could also add a statement redirecting them to the README file if they want other options, either directly in the credit statement, or in the limitations statement (which directs them to acknowledge based on the credit statement; we could add an "or as listed in the README file").

Reply:
@hjkristenson Looking at what you've pushed up, I think it looks great. :+1: